docs/source/user/aerodyn/driver.rst

```diff
@@ -0,0 +1,497 @@
+
+  
+
+
+**Time-dependent analysis**
+
+Time-dependent analyses are used to vary few standard variables during the simulation.
+The variables are: reference wind speed (`HWndSpeed`), powerlaw exponent (`PLExp`), rotor speed (`RotSpd`), collective pitch (`Pitch`), and nacelle yaw (`Yaw`).
+The time series of each variables are provided in a CSV file (`TimeAnalysisFileName`).
+Time-dependent analyses are selected using `AnalysisType=2`. They are restricted to one turbine (`numTurbines=1`).
+
+.. code:: 
+
+    ----- Time-dependent Analysis [used only when AnalysisType=2 and numTurbines=1] ------
+    "TimeSeries.csv" TimeAnalysisFileName - Filename containing time series (6 column: Time, HWndSpeed, PLExp, RotSpd, Pitch, Yaw). 
+
+
+
+
+**Combined-case analyses**
+
+Combined-case analyses are used to run a parametric studies in one single run.
+They are selected using `AnalysisType=3`, and are restricted to one turbine (`numTurbines=1`).
+The variables that can be changed for each simulation are: reference wind speed (`HWndSpeed`),  powerlaw exponent (`PLExp`), rotor speed (`RotSpd`), collective pitch (`Pitch`), nacelle yaw (`Yaw`), time step (`DT`), simulation time (`TMax`), and sinusoidal motion parameters (degree of freedom, `DOF`, amplitude and frequency).
+When `DOF=0`, the turbine base is fixed.
+
+
+.. code::
+
+    ----- Time-dependent Analysis [used only when AnalysisType=3 and numTubines=1] ------
+             4  NumCases     - Number of cases to run
+    HWndSpeed  PLExp   RotSpd   Pitch   Yaw    dT      Tmax   DOF   Amplitude  Frequency 
+    (m/s)      (-)     (rpm)    (deg)  (deg)   (s)     (s)    (-)    (-)       (Hz)
+       8.      0.0       6.     0.      0.     1.0     100     0      0         0.0
+       8.      0.0       6.     0.      0.     1.0     100     0      0         0.0
+       9.      0.1       7.     1.      0.     0.5      50     1      5.0       0.1 
+       9.      0.2       8.     2.      0.     0.5      50     1      2.0       0.2 
+
+
+**Outputs**
+
+The output section controls the format of the tabular output file and VTK files, similar to the OpenFAST outputs.
+The user can control the hub radius and nacelle dimension for the VTK visualization. The nacelle is represented as a sphere of radius (`VTKHubRad`), and the nacelle with a parallelepiped defined using an origin and three lengths parallel to the nacelle coordinates (`VTKNacDim`).
+
+
+.. code::
+
+    ----- Output Settings -------------------------------------------------------------------
+      "ES15.8E2"     OutFmt      - Format used for text tabular output, excluding the time channel.  Resulting field should be 10 characters. (quoted string)
+    2                OutFileFmt  - Format for tabular (time-marching) output file (switch) {1: text file [<RootName>.out], 2: binary file [<RootName>.outb], 3: both}
+    0                WrVTK       - VTK visualization data output: (switch) {0=none; 1=animation}
```

Comment:
Perhaps this doesn't matter, but OpenFAST uses WrVTK = 1 for initialization data only and 2 for animation

Reply:
changed to match openfast convention now